### ISSUE TITLE
Why is RegLoss rising？

### ISSUE THREAD

**Issue by ronyuzhang** [davidsandberg/facenet#548]
Epoch: [89][341/1000]	Time 0.237	Loss 1.267	RegLoss 0.705
Epoch: [89][342/1000]	Time 0.232	Loss 1.202	RegLoss 0.705
Epoch: [89][343/1000]	Time 0.224	Loss 1.178	RegLoss 0.705
Epoch: [89][344/1000]	Time 0.239	Loss 1.233	RegLoss 0.705
Epoch: [89][345/1000]	Time 0.234	Loss 1.204	RegLoss 0.705
Epoch: [89][346/1000]	Time 0.233	Loss 1.257	RegLoss 0.705
Epoch: [89][347/1000]	Time 0.227	Loss 1.246	RegLoss 0.705
Epoch: [89][348/1000]	Time 0.236	Loss 1.248	RegLoss 0.705
Epoch: [89][349/1000]	Time 0.231	Loss 1.255	RegLoss 0.705
Epoch: [89][350/1000]	Time 0.243	Loss 1.237	RegLoss 0.705
Epoch: [89][351/1000]	Time 0.239	Loss 1.198	RegLoss 0.705
Epoch: [89][352/1000]	Time 0.238	Loss 1.280	RegLoss 0.705
Epoch: [89][353/1000]	Time 0.224	Loss 1.302	RegLoss 0.705
Epoch: [89][354/1000]	Time 0.249	Loss 1.258	RegLoss 0.705
Epoch: [89][355/1000]	Time 0.234	Loss 1.243	RegLoss 0.705
Epoch: [89][356/1000]	Time 0.227	Loss 1.219	RegLoss 0.705
Epoch: [89][357/1000]	Time 0.258	Loss 1.268	RegLoss 0.705
Epoch: [89][358/1000]	Time 0.223	Loss 1.207	RegLoss 0.705
Epoch: [89][359/1000]	Time 0.238	Loss 1.228	RegLoss 0.705
Epoch: [89][360/1000]	Time 0.235	Loss 1.260	RegLoss 0.705
Epoch: [89][361/1000]	Time 0.239	Loss 1.220	RegLoss 0.705
Epoch: [89][362/1000]	Time 0.220	Loss 1.225	RegLoss 0.705
Epoch: [89][363/1000]	Time 0.241	Loss 1.215	RegLoss 0.705
Epoch: [89][364/1000]	Time 0.234	Loss 1.249	RegLoss 0.705
Epoch: [89][365/1000]	Time 0.242	Loss 1.296	RegLoss 0.705
Epoch: [89][366/1000]	Time 0.232	Loss 1.213	RegLoss 0.705
Epoch: [89][367/1000]	Time 0.240	Loss 1.243	RegLoss 0.705
Epoch: [89][368/1000]	Time 0.232	Loss 1.217	RegLoss 0.706
Epoch: [89][369/1000]	Time 0.246	Loss 1.280	RegLoss 0.706
Epoch: [89][370/1000]	Time 0.235	Loss 1.181	RegLoss 0.706
Epoch: [89][371/1000]	Time 0.245	Loss 1.225	RegLoss 0.706
Epoch: [89][372/1000]	Time 0.242	Loss 1.228	RegLoss 0.706
Epoch: [89][373/1000]	Time 0.227	Loss 1.276	RegLoss 0.706
Epoch: [89][374/1000]	Time 0.235	Loss 1.235	RegLoss 0.706
Epoch: [89][375/1000]	Time 0.244	Loss 1.272	RegLoss 0.706
Epoch: [89][376/1000]	Time 0.232	Loss 1.237	RegLoss 0.706
Epoch: [89][377/1000]	Time 0.231	Loss 1.234	RegLoss 0.706
Epoch: [89][378/1000]	Time 0.230	Loss 1.318	RegLoss 0.706
Epoch: [89][379/1000]	Time 0.225	Loss 1.234	RegLoss 0.706
Epoch: [89][380/1000]	Time 0.239	Loss 1.231	RegLoss 0.706
Epoch: [89][381/1000]	Time 0.233	Loss 1.226	RegLoss 0.706

**Comment by ronyuzhang** [comment]
@davidsandberg

**Comment by qixianbiao** [comment]
It is normal. Focus on your loss function. According to my experience, if you use one good loss function, the loss can decrease to around 0.10 instead of 1.231.

See my provided three running logs
https://github.com/qixianbiao/PolyFace

**Comment by cactuslei** [comment]
@qixianbiao here the "Loss" already include the "RegLoss", so the cross entropy loss is around 0.5, right?
